packages/admin/cms-admin/src/dam/FileForm/EditFile.tsx

```diff
@@ -87,9 +87,9 @@ const EditFile = ({ id, contentScopeIndicator }: EditFormProps) => {
                             defaultMessage="Failed to load file. <link>Go to Assets</link>"
                             values={{
                                 link: (chunks) => (
-                                    <Link to={`${scopeMatch.url}/assets`} component={RouterLink}>
+                                    <RouterLink style={{ outline: "5px solid green" }} to={`${scopeMatch.url}/assets`}>
```

Comment:
I suppose this was used for debugging 😄 

```suggestion
                                    <RouterLink to={`${scopeMatch.url}/assets`}>
```